packages/arb-token-bridge-ui/src/util/TokenUtils.ts

```diff
@@ -159,14 +168,28 @@ export type FetchErc20AllowanceParams = FetchErc20DataProps & {
  */
 export async function fetchErc20Allowance(params: FetchErc20AllowanceParams) {
   const { address, provider, owner, spender } = params
-
-  // todo: fall back if there is no multicall?
-  const multiCaller = await MultiCaller.fromProvider(provider)
-  const [tokenData] = await multiCaller.getTokenData([address], {
-    allowance: { owner, spender }
-  })
-
-  return tokenData?.allowance ?? constants.Zero
+  try {
+    const multiCaller = await MultiCaller.fromProvider(provider)
+    const [tokenData] = await multiCaller.getTokenData([address], {
+      allowance: { owner, spender }
+    })
+    return tokenData?.allowance ?? constants.Zero
+  } catch (e) {
+    //  fall back if there is no multicall
+    const erc20 = ERC20__factory.connect(address, provider)
+    try {
+      const allowance = await erc20.allowance(owner, spender)
+      return allowance ?? constants.Zero
+    } catch (error) {
+      // contract is not a valid Erc20 token
+      Sentry.configureScope(function (scope) {
+        scope.setExtra('token_address', address)
+        Sentry.captureException(error)
+      })
+
+      return constants.Zero // return fallback value
+    }
+  }
```

Comment:
I'm not so sure if we should add catch here and return a value if everything fails. I'd rather we try/catch this in the UI and then display a toast. Same thing for `fetchErc20Data`